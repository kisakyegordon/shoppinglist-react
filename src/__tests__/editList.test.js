// import React from 'react';
// import {shallow} from 'enzyme';
// import {shallowToJson} from 'enzyme-to-json';
// import EditList from '../components/edit/editList';


// let match = {
//     params : '1'
// }
// let res = {
//     body : {
//         list : {
//             name : "ghjg"
//         }
//     }
// }
// let name = "sdfasd"
// console.log(res)
// const setUp = () => {
//     const props = {
//         match,
//         name
//     };
//     return shallow(<EditList {...props} />);
// };


// describe('<EditList />', () => {
//     it('match a snapshot', () => {
//         const wrapper = setUp();
//         expect(shallowToJson(wrapper)).toMatchSnapshot();
//     });
// });