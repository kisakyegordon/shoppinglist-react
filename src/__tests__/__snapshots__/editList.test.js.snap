// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EditList /> match a snapshot 1`] = `
<div
  className="signin"
  style={
    Object {
      "marginTop": "75px",
      "minHeight": "200px",
    }
  }
>
  <h3>
     Edit List Name 
  </h3>
  <form
    onSubmit={[Function]}
  >
    <div
      style={
        Object {
          "display": "inline-block",
          "marginBottom": "30px",
        }
      }
    >
      <TextField
        disabled={false}
        floatingLabelFixed={false}
        fullWidth={false}
        multiLine={false}
        name="name"
        onChange={[Function]}
        placeHolder=""
        rows={1}
        type="text"
        underlineShow={true}
        value=""
      />
    </div>
    <div
      className="Gaga"
    >
      <RaisedButton
        disabled={false}
        fullWidth={false}
        label="Cancel"
        labelPosition="after"
        onClick={[Function]}
        primary={false}
        secondary={false}
      />
      <RaisedButton
        disabled={false}
        fullWidth={false}
        icon={<ContentSave />}
        label="Save"
        labelPosition="after"
        primary={true}
        secondary={false}
        type="submit"
      />
    </div>
  </form>
</div>
`;
